test/fixtures/cdb_crankshaft/cdb_voronoi.sql

```diff
@@ -0,0 +1,15 @@
+CREATE SCHEMA IF NOT EXISTS cdb_crankshaft;
```

Comment:
What about having the same approach as with cdb_dataservices_client? Having a file for the schema.